reactjs-films-homework/src/components/MovieCard/MovieCard.tsx

```diff
@@ -0,0 +1,43 @@
+import { Box, Paper, Stack, Typography } from '@mui/material';
+import { useStyles } from './styles';
+
+type MovieCardProps = {
+  popularity: string;
+  id: number;
+  imageHeight: string;
+  isDescription?: boolean;
+  onClick?: () => void;
+};
+
+export const MovieCard = ({
+  popularity,
+  id,
+  imageHeight,
+  isDescription = true,
+  onClick
+}: MovieCardProps) => {
+  const classes = useStyles({ imageHeight });
```

Comment:
as I see for this component you use only 400px move this to styles and get rid from this props, also see ActorCard